gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.1-bin.zip
```

Comment:
It will be great to use Gradle 6+ from start if we can

Reply:
Done. That's a very good point. I had to change plugin resolution a bit, due to Gradle changes, but it works :). I will also bump up all of the libraries in another PR.